### ISSUE TITLE
list-item-content-indent issue with indented code blocks

### ISSUE THREAD

**Issue by EvgenyOrekhov** [remarkjs/remark-lint#111]
I have the following markdown:

```md
1. **Fork**
2. **Clone**

        git clone <fork_url>
3. **Configure remotes**

        cd <project_dir>
        git remote add upstream <upstream_dir>
```

remark-lint says "Don’t use mixed indentation for children, remove 1 space", but those are not children, those are indented code blocks, and they must be indented with 8 spaces.

**Comment by wooorm** [comment]
Hi @EvgenyOrekhov, thanks for using remark-lint.

You’re using 8 spaces of indentation for code, which makes them start at 4, whereas the paragraphs start at 3. Thus the warning.

Makes sense?

**Comment by EvgenyOrekhov** [comment]
@wooorm I can't use 7 spaces for code because it won't be rendered as code, it renders correctly only if I use 8 spaces. Besides, when I use 7 spaces in this example, remark-lint still keeps saying "Don’t use mixed indentation for children, remove 1 space", which makes no sense.

**Comment by wooorm** [comment]
Ah, the solution I’d go with is increase the indent of paragraphs (see [list-item-indent](https://github.com/wooorm/remark-lint/blob/master/doc/rules.md#list-item-indent)). That rule is in place as otherwise you’ll run into differences between markdown implementations for lists (especially when there’s code too, which is what’s happening here!).

**Comment by wooorm** [comment]
@EvgenyOrekhov Did that help? Can I close this?

**Comment by EvgenyOrekhov** [comment]
@wooorm I want to use the following config:

```json
{
    "listItemContentIndent": true,
    "listItemIndent": "space"
}
```

How should I rewrite my markdown so that there were no warnings?

This example at RunKit: https://runkit.com/57d675ee7580c314003cf648/58468fcb9842660013ba4aee

**Comment by N0rbert** [comment]
Still a problem, got it in the following RMarkdown fragment:

```
1. Clone repository

       git clone https://github.com/rstudio/bookdown-demo.git

1. Enjoy
```

**Comment by wooorm** [comment]
Sorry for the wait! I just wanted to share that there’s now a PR that solves this issue: https://github.com/remarkjs/remark/pull/536.

**Comment by wooorm** [comment]
This is now released in [`remark@13.0.0`](https://github.com/remarkjs/remark/releases/tag/13.0.0)
